integration_tests/test_multiarch_stage_packages.py

```diff
@@ -31,6 +31,6 @@ def test_bad_multiarch_stage_packages(self):
         self.assertThat(
             exception.output, Contains(
                 "Error downloading stage packages for part 'my-part': The "
-                "Ubuntu package 'hello:fake-arch' was not found.\n"
+                "package 'hello:fake-arch' was not found.\n"
```

Comment:
This is apt-ism, when dnf is implemented this test should just be expanded to behave differently depending on the platform it runs on, the error message is generated generically though